src/core/src/device/command.rs

```diff
@@ -1,41 +1,32 @@
-// Copyright 2014 The Gfx-rs Developers.
```

Comment:
Ah git must've recognized the file as being moved because of the similarities between the files, whereas I actually copied some things from `draw.rs` and I didn't think about the licenses. Will add the the license headers.